docs/BUILDING_INSTRUCTIONS.md

```diff
@@ -0,0 +1,111 @@
+# Solution building instructions
+
+This guide explains how to:
+
+* Install required software
+* Clone the repository (source code) to your local computer
+* Build (compile) the source code to DLL files
+* Test in-game
+
+We strongly recommend doing development work on Windows as there's much more tooling available. If you're using a Mac, set up a Boot Camp partition for Windows and use that. If you're using Linux, consult the internets.
+
+## Install
```

Comment:
Somewhere around this section you can mention that the solution expects to find `TLM/dependencies` directory with the game C# DLLs copied into it, or a link to `<Steam>/Cities_Skylines/Cities_Data/Managed`.

Reply:
The reference path is mentioned but I didn't know about the `TLM/depedencies` folder, will add that in.

Reply:
If you symlink it before trying the first build, it just finds everything and builds and never requires any references manipulation.

Reply:
@kvakvs I've updated build docs to include note on the `TLM\dependencies` folder